clouddriver-openstack/clouddriver-openstack.gradle

```diff
@@ -14,8 +14,13 @@
  * limitations under the License.
  */
 
+repositories {
+  maven { url "https://oss.sonatype.org/content/repositories/snapshots" }
```

Comment:
ok with this in the short-term but as soon as possible it would be great to get this out of here